### DISCUSSION TITLE
How to set configuration options?

### DISCUSSION THREAD

**Discussion by amca01** [KaTeX/KaTeX#2714] [Q&A] [answered]
Hello

I am blogging using the Hugo static platform, and I have just changed over to a new theme which supports KaTeX.  As far as I can tell, my needs are such that KaTeX will well meet them all.  However, I'm not sure how - or rather, where - to place configuration options.  The theme subdirectory contains a file called `math.html` which simply consists of scripts which call on the KaTeX renderer and minified package as JavaScript and CSS files from the KaTeX repository:

    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css" integrity="sha384-AfEj0r4/OFrOo5t7NnNe46zW/tFgW6x/bCJG8FqQCEo3+Aro6EYUG4+cU+KJWu/X" crossorigin="anonymous">
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js" integrity="sha384-g7c+Jr9ZivxKLnZTDUhnkOnsh30B4H0rpLUpJ4jAIKs4fnJI+sEnkvrMWph2EDg4" crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js" integrity="sha384-mll67QQFJfxn0IYznZYonOWZ644AWYC+Pt2cHqMaRhXVrursRwvLnLaebdGIlYNa" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script> 

These are the only instances of KaTeX being called in the subdirectory.  Given this setup, where or how do I place configuration options?  I should say that all I want to do is to set `fleqn: true` as a global option.

Thank you very much!

**Comment by k4b7** [comment, marked as answer]
https://katex.org/docs/autorender.html provides documentation on how to provide options to `renderMathInElement`.

**Reply by edemaine** [reply]
And note that `renderMathInElement` is called at the very end of the last line you included; you'd provide options after `document.body`.

**Reply by amca01** [reply]
Thank you both for the pointer - yes indeed, once I knew where to actually make that change it was a very quick fix - and works perfectly!
